bitcoin-units/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "bitcoin-units"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+homepage = "https://github.com/rust-bitcoin/rust-bitcoin/"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+documentation = "https://docs.rs/bitcoin/"
+description = "Bitcoin units"
+keywords = ["crypto", "bitcoin"]
```

Comment:
Obligatory "crypto means cryptography" and there is no cryptography in the crate. I'd add "newtypes".

Reply:
Ha, It hurt to write this line. I'll remove it from all our crates :)

Reply:
Used "bitcoin" and "newtypes"